### PR TITLE
Update .NET Core 3.1

### PR DESCRIPTION
**Pull request by razfriman**
Updated to .NET Core 3.1

Added GitHub Actions CI - Sample Run: https://github.com/razfriman/MapleShark/actions/runs/158032929/workflow

---

TODO: Fix ContextMenuStrip event

### CODE REVIEW

**Comment by razfriman** [comment]
@diamondo25 - Are you still maintaining this project?

File: SyntaxBox/Design/ComponaCollectionEditor.cs [hunk 1]
```diff
@@ -91,10 +91,11 @@ public ComponaCollectionForm(CollectionEditor e) : base(e)
                 if (types.Length > 1)
                 {
                     GUI.btnDropdown.Visible = true;
-                    GUI.btnDropdown.ContextMenu = new ContextMenu();
+                    GUI.btnDropdown.ContextMenuStrip = new ContextMenuStrip();
                     for (int i = 0; (i < types.Length); i ++)
                     {
-                        GUI.btnDropdown.ContextMenu.MenuItems.Add(new TypeMenuItem(types[i], btnDropDownMenuItem_Click));
+                        //GUI.btnDropdown.ContextMenuStrip.Items.Add(types[i].Name, null, btnDropDownMenuItem_Click);
+                        GUI.btnDropdown.ContextMenuStrip.Items.Add(types[i].Name);
```
Comment (razfriman) [inline SyntaxBox/Design/ComponaCollectionEditor.cs:98]: TODO - Fix this part

File: SyntaxBox/Design/ComponaCollectionEditor.cs [hunk 2]
```diff
@@ -167,15 +168,15 @@ protected void btnUp_Click(object o, EventArgs e)
...
-                var tmi = o as TypeMenuItem;
-                if (tmi != null) 
-                    CreateAndAddInstance(tmi.Type as Type);
-            }
+            //protected void btnDropDownMenuItem_Click(object o, EventArgs e)
+            //{
+            //    var tmi = o as TypeMenuItem;
+            //    if (tmi != null) 
+            //        CreateAndAddInstance(tmi.Type as Type);
+            //}
```
Comment (razfriman) [inline SyntaxBox/Design/ComponaCollectionEditor.cs:180]: Fix this

**Comment by diamondo25** [comment]
Not really maintaining no, due to the loss of interest in Maple (also rip Maple2)

**Pull request closed without merging**